src/cryptext/plugin/simple.py

```diff
@@ -0,0 +1,38 @@
+import os
+
+from cryptext.password import PasswordData, PasswordDataIO
+
+# @dataclass
+# class PasswordData:
+#    label: str
+#    url: str
+#    com: str
+#    user: str
+#    passwd: str
+
+
+with open('test/test.txt', 'r') as file:
+    lines = file.read().splitlines()
+
+content = [{}]
+
+for l in lines:
+    c = [x.strip() for x in l.split('=')]
+
+    if len(c) == 1:
+        content.append({})
+    else:
+        content[-1][c[0]] = c[1]
+
+
+pass_list = []
+for c in content:
+    buffer = PasswordData()
+    for key, value in c.items():
+        setattr(buffer, key, value)
```

Comment:
If the content does contain all mandatory attributes (e.g. the label if it does not have a default value in the `PasswordData` class), the inner loop can be replaced by:
``` python
buffer = PasswordData(**c)
```
The `**c` expands the dictionary items into keyword arguments automatically.

Reply:
In case some mandatory argument is missing, the above code will fail. But it can be catch with an `except AttributeError` quite easily to handle the case if needed.

Reply:
This part is still available if we manage with unfilled attributes ?